ccdaservice/ssmanager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+$ignoreAuth = true;
+// require_once(dirname(__file__) . './../interface/globals.php');
+require_once ( dirname( __FILE__ ) . "/../library/log.inc" );
+require_once ( dirname( __FILE__ ) . "/../library/sql.inc" );
+set_time_limit( 0 );
+ob_implicit_flush();
+session_write_close(); // Release session lock to prevent freezing of other scripts
+ignore_user_abort( 1 );
```

Comment:
Is there a more strategic place for these flags (in case wish to use this library without starting session_write_close() )?